src/ReactiveUI/Platforms/netcoreapp1.0/PlatformRegistrations.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MS-PL license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reactive.Concurrency;
+
+namespace ReactiveUI
+{
+    public class PlatformRegistrations : IWantsToRegisterStuff
+    {
+        public void Register(Action<Func<object>, Type> registerFunction)
+        {
+            RxApp.TaskpoolScheduler = TaskPoolScheduler.Default;
```

Comment:
* It looks like it is. Should I remove that setting from all of the platforms?
* I'll set it to `DefaultScheduler.Instance`

The only place that logs a warning about a missing `IPlatformOperations` looks to be in `RoutedViewHost`, which is only available in builds using the `Platforms/windows-common` files.